app/src/main/res/values/styles_v2.xml

```diff
@@ -184,10 +184,11 @@
   </style>
 
   <style name="Clinic.V2.TextAppearance.Body2Tag">
-    <item name="android:textSize">12sp</item>
+    <item name="android:textSize">14sp</item>
```

Comment:
Were the designs updated for this? Please link to Zeplin.

Reply:
Yup, they were updated. I can't link to a Zeplin component, but the screen is [HERE](https://zpl.io/agkOWA0). You should  be able to find the component from there.